.gitattributes

```diff
@@ -0,0 +1,11 @@
+* text=auto eol=lf
+
+*.js text
```

Comment:
We should also add:
- *.txt
- *.yml
- *.sql 
Just to be thorough and cover our travis yml, a text test file and some .sql files we use for Postgres. Wouldn't want anything unexpected to sneak through on those.

Reply:
Good idea. Added